controllers/fenceagentsremediation_controller_test.go

```diff
@@ -218,6 +357,10 @@ func getFenceAgentsRemediation(nodeName, agent string, sharedparameters map[v1al
 			Agent:            agent,
 			SharedParameters: sharedparameters,
 			NodeParameters:   nodeparameters,
+			// Set the retry count to the minimum for the majority of the tests
```

Comment:
Similar to e2e tests- please set these vars as Const in the beginning

Reply:
I am unsure of the benefits. These values are only used here and it seems to me that their meaning is already clear without a specific name.

Reply:
ok